content/layout/intrinsic-4_grouped-listview/index.md

```diff
@@ -0,0 +1,404 @@
+---
+page_type: guide
+title: Grouped ListView Layout
+author: adamargyle
+description:
+tags:
+- Layout
+- Design
+- CSS
+web_lighthouse: N/A
+web_updated_on:
+web_published_on:
+wf_blink_components: Blink>Accessibility
+---
+
+<figure style="text-align:center; margin: 4rem 0;">
+  <img src="Grouped-listview.png" alt="" class="screenshot">
+</figure>
+
+I call this a grouped listview because **it's a list of lists**. The first list is a vertical list, then each item in that list has a horizontal list! **2-dimensional array UI!**
+
+
+<div style="text-align:center; font-size: 6rem; margin: 10rem 0;">
+  🤯
+</div>
+
+Here's a **less noisy visual** of a list of lists:
+
+<figure style="text-align:center; margin: 2rem 0;">
+  <img src="Grouped-listview multi.png" alt="">
+</figure>
+
+**in JavaScript**
+```js
+const multidimensional = [
+  [1,2,3,4],
+  [1,2],
+  [1],
+  [1,2,3,4,5],
+  [1,2,3],
+]
+```
+
+<div class="note">a list of lists, a collection of collections!</div>
+
+<br><br><br>
+
+
+##### We got this!
+# The Vertical List
+
+<figure style="text-align:center; margin: 4rem 0;">
+  <img src="groupedlist-highlighted.png" alt="" class="screenshot">
+</figure>
+
+What are we lookin at **in terms of layout**?
+- Rows with **variable height**
+- Big **healthy gaps**
+- We **didn't** put the section **title in a separate row**
+- Let's use `<section>` elements for these list items
+
+<br><br>
+
+**HTML**
+```html
+<article>
+  <section>...</section>
+  <section>...</section>
+  <section>...</section>
+  <section>...</section>
+</article>
+```
+
+<div class="note">
+  <b>Note:</b> This grouped listview is inside of an <code>article</code> tag, it's semantically nice to have <code>main > article > section(s)</code> in our HTML
+</div>
+
+<br><br>
+
+**CSS**
+```css
+article {
+  display: grid;
+  gap: 5rem;
+}
+```
+
+<div class="note">
+  <b>CSS Grid In English:</b> We're <b>leaning into grid defaults and it's intrinsic layout behavior</b> here and just specifying the gaps.
+</div>
+
+<br><br>
+
+**Glitch**
+<div class="glitch-embed-wrap" style="height: 346px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/logical-tab-order?path=index.html&previewSize=100&attributionHidden=true"
+    alt="logical-tab-order on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+<br><br><br><br>
+
+
```

Comment:
Calling amnesty on review until Glitches are in play! This is a hefty section, and I kind of want to dive into the glitches as part of review.

Reply:
i dont blame ya!